incident_viewer_demo/incident/incident.go

```diff
@@ -65,74 +64,45 @@ func (i *IncidentData) Init(
 	i.GoodPeriodStart = goodTimeStart
 	i.GoodPeriodEnd = goodTimeEnd
 	i.BadPeriodStart = badTimeStart
+	i.BadPeriodEnd = badTimeEnd
 	i.GoodPeriodMetric = goodMetric
 	i.BadPeriodMetric = badMetric
-	i.BadPeriodEnd = badTimeEnd
 	i.Severity = severity
 	i.NumTestsAffected = testsAffected
 	i.GoodPeriodInfo = goodPeriodInfo
 	i.BadPeriodInfo = badPeriodInfo
 	i.IncidentInfo = incidentInfo
 }
 
-func (i *DefaultIncident) Init(goodTimeStart time.Time, goodTimeEnd time.Time,
+/* Assign data members and set appropriate text for information fields */
+func (i *DefaultIncident) MakeIncidentData(goodTimeStart time.Time, goodTimeEnd time.Time,
 	badTimeStart time.Time,
 	badTimeEnd time.Time,
 	avgDSGood float64,
 	avgDSBad float64,
-	severity float64,
+	severityDecimal float64,
```

Comment:
is there a reason this is switching between severity and severityDecimal?

Reply:
I personally like severity more, it's already implied that it's a decimal since it's a float

Reply:
Changed to severity